v2/AdminCore.Common/Interfaces/IDashboardService.cs

```diff
@@ -9,7 +9,7 @@ public interface IDashboardService
   {
     IList<EventDto> GetEmployeeDashboardEvents(int employeeId, DateTime date);
 
-    IList<EventDto> GetEmployeeEvents(int employeeId, DateTime date);
+    IList<EventDto> GetEmployeeEventsForMonth(int employeeId, DateTime date);
```

Comment:
EmployeeId is no longer being passed into the Services, instead we're using AuthenticatedUser injection. You can see this in the HolidayController/EventService

Reply:
Authenticated user is being injected by the controller, employeeId is included in this service method for re-usability.